apps/web/src/api/layouts.ts

```diff
@@ -4,10 +4,18 @@ export async function createLayout(data: any) {
   return api.post(`/v1/layouts`, data);
 }
 
+export async function updateLayout(layoutId: string, data: any) {
```

Comment:
We should do proper typing for data.
What I'd suggest is that I can add `LayoutDto` to the `/lib/shared` so then you can rely on it to do a typing like:
```
type UpdateLayoutRequestDto = Pick<LayoutDto, 'name' | 'description' | 'variables' | 'content' | 'isDefault'>;
```

Another option is make shared the current `UpdateLayoutRequestDto` from the API so you can reuse it.
What would be your preference?
```

Reply:
It will also be needed for the create layout. I think it will be better to have the dtos in shared. Wdyt?